app/admin/participant.rb

```diff
@@ -35,7 +35,10 @@
 
     table_for Datastream.where(participant_id: participant.id) do
       column 'Datasource', :datasource_id do |ds|
-        Datasource.find_by_id(ds).display_name
+        datas = Datasource.find_by_id(ds)
+        if datas.present?
```

Comment:
__[rubocop]__
* Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`. [:ref](https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier)